webapp/security/api.py

```diff
@@ -113,6 +113,42 @@ def get_notices(
 
         return notices_response.json()
 
+    def get_page_notices(
```

Comment:
If we switch the remaining views `single_notices_sitemap`, `notices_sitemap`, and `notices_feed` to consume this new endpoint we can remove the old `get_notices` function completely

Reply:
Alright, I'll switch them over

Reply:
Done!